mappings/net/minecraft/util/math/MatrixUtil.mapping

```diff
@@ -0,0 +1,14 @@
+CLASS net/minecraft/unmapped/C_bmzqkdhp net/minecraft/util/math/MatrixUtil
+	FIELD f_arxnkhow G F
+	FIELD f_ddnzxrmh CS F
+	FIELD f_kqccjreh SS F
+	METHOD m_ibrymiig svdDecompose (Lorg/joml/Matrix3f;)Lorg/apache/commons/lang3/tuple/Triple;
+		ARG 0 matrix3f
+	METHOD m_jeovkadj toAffine (Lorg/joml/Matrix4f;)Lorg/joml/Matrix4x3f;
+		ARG 0 matrix4f
+	METHOD m_kcqrzkbi approxGivensQuat (FFF)Lcom/mojang/datafixers/util/Pair;
+	METHOD m_kniviwwf qrGivensQuat (FF)Lcom/mojang/datafixers/util/Pair;
+	METHOD m_skxdryqk stepJacobi (Lorg/joml/Matrix3f;)Lorg/joml/Quaternionf;
+		ARG 0 matrix3f
+	METHOD m_xockxffa mulComponentWise (Lorg/joml/Matrix4f;F)Lorg/joml/Matrix4f;
+		ARG 0 matrix4f
```

Comment:
I don't think these names are really good... maybe you could use or adapt yarn's names for these methods?